cove_iati/rulesets/iati_standard_v2_ruleset/steps/standard_ruleset_step_definitions.py

```diff
@@ -26,28 +27,28 @@ def step_given_elements(context, xpath_expression):
     context.xpath_expression = xpath_expression
 
 
-@given('`{xpath_expression1}` plus `{xpath_expression2}`')
-def step_given_two_different_elements(context, xpath_expression1, xpath_expression2):
-    context.xpath_expression1 = xpath_expression1
-    context.xpath_expression2 = xpath_expression2
-
-
-@then('`{attribute}` attribute must be today or in the past')
+@then('`{xpath}` must be today or in the past')
 @register_ruleset_errors()
-def step_must_be_today_or_past(context, attribute):
+def step_must_be_today_or_past(context, xpath):
     errors = []
     today = datetime.date.today()
+    parents = get_xobjects(context.xml, context.xpath_expression)
```

Comment:
So if the ```context.xpath_expression``` is ```'.'``` (which you have changed in few places) then we just get ```context.xml``` back within a array, right?

Reply:
Yes